timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -82,7 +82,9 @@
 
     private static final int DEFAULT_LOCK_TIMEOUT_MS = 10_000;
     private static final int LONG_LOCK_TIMEOUT_MS =
```

Comment:
Maybe rename this

Reply:
Yep, renamed to `LONGER_THAN_READ_TIMEOUT_LOCK_TIMEOUT_MS`